lib/bundler/settings.rb

```diff
@@ -256,5 +280,11 @@ def normalize_uri(uri)
       uri
     end
 
+    def print_remembered_flags_deprecation_warning
+      Bundler.ui.warn "Starting in Bundler 2.0, flags passed into commands will no longer be remember past that command." \
+        " Instead please set flags you want remembered between commands using `bundle config <setting name> <setting value>`" \
+        " To ignore this warning use `bundle config ignore_warnings.remembered_flags true`"
+      set_current "ignore_warnings.remembered_flags", true
```

Comment:
Considering https://github.com/bundler/bundler/pull/3590/files, is this change necessary? If so, shouldn't it be printed as a deprecation warning in 1-99-dev rather than as a ui.warn in 2-0-dev?